src/pages/components/callout-with-media.mdx

```diff
@@ -23,9 +24,19 @@ Callout &mdash; with media uses high contrast colors to create a delightful paus
 
 ![Default](../../images/pattern/callout-with-media/callout-with-media.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes   | 1 | XXS (40 / 55)|
+| Paragraph | Long copy | No  | 1 | XXL (350 / 450) |
+| [Video player](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/video-player) or [Image with caption](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image-with-caption) | Component | Yes | 1 | - |
```

Comment:
Should be the same as `content block — simple` if we'd like to link out. If you decide to maintain images should be allowed here as well.

Reply:
@photodow I'm wondering if we should just mention below the table that this component refers to "content block - simple". That way we are referring to it, but the information is also here. Thoughts?

Reply:
That might work for now, but I'm not sure it gets at the crux of maintaining the same data in multiple places.